k8s/regions/oregon-b/stage.yaml

```diff
@@ -134,3 +134,7 @@ app:
     zendesk_api_token: SECRET
     zendesk_user_email: SECRET
     zendesk_subdomain: SECRET
+    zendesk_category_field_id: 360047206172
+    zendesk_os_field_id: 360018604871
+    zendesk_vpn_field_id: 360047198211
+    zendesk_ticket_form_id: 360000417171
```

Comment:
Only the `ticket_form_id` ought to change between stage/prod, so can we not set the `*_field_id` variables - and pick up the defaults from settings.py - until we definitely need them?

Reply:
I will go with better explicit that implicit

Reply:
With such similar IDs across all the fields (it's hard to glance at them and immediately notice the difference) I think being explicit just makes it more confusing. Could these at least be ordered in the same way as the fields in the form/settings to make comparing them top-to-bottom easier.